cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -16,8 +16,13 @@ You can also find the URL to the latest binaries from the {product-title} web co
 
 include::modules/developer-cli-odo-installing-odo-on-linux.adoc[leveloffset=+1]
 
+include::modules/developer-cli-odo-installing-odo-on-linux-on-IBM-Power.adoc[leveloffset=+1]
+
+<<<<<<< HEAD
```

Comment:
This `<<<<<<< HEAD` text comes from the merge conflict. It does not need to be in the final file. Please render the file to look as you would like without reference to merge conflict suggestion syntax.

Reply:
This is also impacting the rendered preview.